src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationAttribute.kt

```diff
@@ -59,14 +60,38 @@ class LocationAttribute(
   @OneToMany(mappedBy = "locationAttribute", fetch = FetchType.LAZY, cascade = [CascadeType.ALL], orphanRemoval = true)
   private val locationSchedule: MutableList<LocationSchedule> = mutableListOf()
 
-  fun setLocationSchedule(schedules: List<LocationSchedule>) {
+  fun schedule() = locationSchedule.toList()
+
+  fun isLocationUsage(usage: LocationUsage) = locationUsage == usage
+
+  fun addSchedule(
```

Comment:
There is potentially further validation to add here e.g. we don't want to allow duplicate entries.